.github/workflows/release.yml

```diff
@@ -26,6 +26,12 @@ on:
         required: true
         default: false
         type: boolean
+      assets:
+        description: "Path expression for Assets to upload."
+        type: string
+        required: false
+        default: "cli/neo4jv?/target/*-distribution.zip"
```

Comment:
The default value should be `target/checkout/cli/neo4jv?/target/*-distribution.zip`